scripts/create-database.js

```diff
@@ -0,0 +1,57 @@
+// scripts/create-database.js
```

Comment:
Theres no need for the commented name of the file here, it just creates extra noise so it should be removed